rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -1714,7 +1723,7 @@ public String taskLogByTag(@HeaderParam("sessionid") String sessionId, @PathPara
             @PathParam("tasktag") String taskTag)
             throws NotConnectedRestException, UnknownJobRestException, PermissionRestException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)